### ISSUE TITLE
Uncaught TypeError: marked.setOptions is not a function

### ISSUE THREAD

**Issue by ajboni** [sparksuite/simplemde-markdown-editor#828]
I think it might be related to #825

**Comment by staszek998** [comment]
You're right, @ajboni - it's related. That's the exact same `TypeError` that's caused by the `marked` package in version `4.x`. When the @sparksuite will implement my PR into their codebase, the `TypeError` you mentioned will be gone.

**Comment by ajboni** [comment]
Thanks for confirming, @staszek998 . For now, a workaround I'm using, is to move to yarn and use `resolutions` on `package.json`. Not really ideal, but it works...

```
  "resolutions": {
    "marked": "3.0.8"
  }
```

**Comment by staszek998** [comment]
I'm using the same workaround and it works for me, too 🙂
